src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -516,11 +516,12 @@ long IslandWindow::_calculateTotalSize(const bool isWidth, const long clientSize
             // and HIWORD treat the coordinates as unsigned quantities.
             const til::point eventPoint{ GET_X_LPARAM(lparam), GET_Y_LPARAM(lparam) };
             // This mouse event is relative to the display origin, not the window. Convert here.
-            const til::rectangle windowRect{ GetWindowRect() };
+            const til::rect windowRect{ GetWindowRect() };
             const auto origin = windowRect.origin();
             const auto relative = eventPoint - origin;
             // Convert to logical scaling before raising the event.
-            const auto real = relative / GetCurrentDpiScale();
+            const auto scale = GetCurrentDpiScale();
+            const til::point real{ til::math::flooring, relative.x / scale, relative.y / scale };

```

Comment:
The previous code didn't specify the exact `til::math::*` rounding mode to use. This one does.
This fixes at least one bug in our code where we use `til::point / float`, which implicitly does a flooring narrow, instead of a more appropriate `til::math::round`.